utils/forms/validations/index.ts

```diff
@@ -24,3 +28,10 @@ export const signUpValidationSchema = Yup.object().shape({
     .oneOf([Yup.ref(password.name)], "Passwords must match")
     .required("Please confirm your password"),
 });
+
+export const signInValidationSchema = Yup.object().shape({
+  email: Yup.string()
+    .email("Invalid email address")
+    .required(`${signInEmail?.label} is required`),
+  password: Yup.string().required(`${signInPassword?.label} is required`),
```

Comment:
apply proper validations for password like the ones in sign up or atleast some of it

Reply:
we need to use that validation here, user is not creating password here. If password is incorrect user is getting error from the backend.

Reply:
Atleast apply the min and max one

Reply:
Applying the minimum and maximum validation is unnecessary as well